### PR TITLE
Next round with ajax

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Fixes #204

**Comment by dionyziz** [comment]
@pkakelas please review these front-end changes.

**Comment by pkakelas** [comment]
@dionyziz I need a screenshot in order to review. @VitSalis pls provide me if possible. If the changes are not noticeable then as I don't see any changes in any abstract selector then it LGTM.

**Comment by vitsalis** [comment]
Before clicking next:
selection_008
after clicking next
selection_009

File: static/script/game/view.js [hunk 1]
```diff
@@ -34,4 +34,59 @@ $( document ).ready( function() {
         $infobubble.removeClass( 'reversed' );
         $infobubble.hide();
     } );
+    function fixRoundId( classname ) {
+        var href = $( '.' + classname + ' a' ).attr( 'href' );
+        var roundid;
+        var attribute;
+        var gameid;
+
+        hrefArray = href.substr( href.indexOf( "?" ) + 1 ).split( "&" );
```
Comment (dionyziz) [inline static/script/game/view.js]: Don't use global variables

File: static/script/game/view.js [hunk 2]
```diff
@@ -34,4 +34,59 @@ $( document ).ready( function() {
...
+        for ( var i = 0; i < hrefArray.length; ++i ) {
+            attribute = hrefArray[ i ].split( "=" );
+            if ( attribute[ 0 ] == 'roundid' ) {
```
Comment (dionyziz) [inline static/script/game/view.js]: Use `switch` to check the value of the same variable.

File: static/script/game/view.js [hunk 3]
```diff
@@ -34,4 +34,59 @@ $( document ).ready( function() {
         $infobubble.removeClass( 'reversed' );
         $infobubble.hide();
     } );
+    function fixRoundId( classname ) {
+        var href = $( '.' + classname + ' a' ).attr( 'href' );
```
Comment (dionyziz) [inline static/script/game/view.js]: Store jQuery object for multiple uses.

File: static/script/game/view.js (hunk 1 above)
Comment (vitsalis) [inline static/script/game/view.js]: I don't think I am.

Comment (dionyziz) [inline static/script/game/view.js]: Where is `hrefArray` defined?

Comment (vitsalis) [inline static/script/game/view.js]: nowhere. I don't know hwy I had no error.

Comment (dionyziz) [inline static/script/game/view.js]: Because undeclared variables become globals.

**Pull request merged**